single.php

```diff
@@ -1,32 +1,50 @@
 <?php
```

Comment:
Why did this file change so much? Is it a page we're not using? If so, should it be removed or have a note at the top of the file?